docker-compose/solr6-override-docker-compose.yml

```diff
@@ -56,6 +56,3 @@ services:
   search-reindexing:
     image: busybox:musl
     command: "true"
-  elasticsearch:
-    image: busybox:musl
```

Comment:
Why we drop this?

Reply:
Audit Storages requires Elasticsearch to store audit logs. In that case, we don't need to disable Elasticsearch in the solr6 docker-compose file.